core/src/main/java/com/sap/ai/sdk/core/model/KpiResultRowItem.java

```diff
@@ -11,123 +11,8 @@
-  @Nonnull
-  public Set<String> getCustomFieldNames() {
-    return cloudSdkCustomFields.keySet();
-  }
-
-  /**
-   * Get the value of an unrecognizable property of this {@link KpiResultRowItem} instance.
-   *
-   * @deprecated Use {@link #toMap()} instead.
-   * @param name The name of the property
-   * @return The value of the property
-   * @throws NoSuchElementException If no property with the given name could be found.
-   */
-  @Nullable
-  @Deprecated
-  public Object getCustomField(@Nonnull final String name) throws NoSuchElementException {
-    if (!cloudSdkCustomFields.containsKey(name)) {
-      throw new NoSuchElementException("KpiResultRowItem has no field with name '" + name + "'.");
-    }
-    return cloudSdkCustomFields.get(name);
-  }
-
-  /**
-   * Get the value of all properties of this {@link KpiResultRowItem} instance including
-   * unrecognized properties.
-   *
-   * @return The map of all properties
-   */
-  @JsonIgnore
-  @Nonnull
-  public Map<String, Object> toMap() {
-    final Map<String, Object> declaredFields = new LinkedHashMap<>(cloudSdkCustomFields);
-    return declaredFields;
-  }
-
-  /**
-   * Set an unrecognizable property of this {@link KpiResultRowItem} instance. If the map previously
-   * contained a mapping for the key, the old value is replaced by the specified value.
-   *
-   * @param customFieldName The name of the property
-   * @param customFieldValue The value of the property
-   */
-  @JsonIgnore
-  public void setCustomField(@Nonnull String customFieldName, @Nullable Object customFieldValue) {
-    cloudSdkCustomFields.put(customFieldName, customFieldValue);
-  }
-
-  @Override
-  public boolean equals(@Nullable final java.lang.Object o) {
-    if (this == o) {
-      return true;
-    }
-    if (o == null || getClass() != o.getClass()) {
-      return false;
-    }
-    final KpiResultRowItem kpiResultRowItem = (KpiResultRowItem) o;
-    return Objects.equals(this.cloudSdkCustomFields, kpiResultRowItem.cloudSdkCustomFields);
-  }
-
-  @Override
-  public int hashCode() {
-    return Objects.hash(cloudSdkCustomFields);
-  }
-
-  @Override
-  @Nonnull
-  public String toString() {
-    final StringBuilder sb = new StringBuilder();
-    sb.append("class KpiResultRowItem {\n");
-    cloudSdkCustomFields.forEach(
-        (k, v) ->
-            sb.append("    ").append(k).append(": ").append(toIndentedString(v)).append("\n"));
-    sb.append("}");
-    return sb.toString();
-  }
-
-  /**
-   * Convert the given object to string with each line indented by 4 spaces (except the first line).
-   */
-  private String toIndentedString(final java.lang.Object o) {
-    if (o == null) {
-      return "null";
-    }
-    return o.toString().replace("\n", "\n    ");
-  }
-
-  /** Create a new {@link KpiResultRowItem} instance. No arguments are required. */
-  public static KpiResultRowItem create() {
-    return new KpiResultRowItem();
-  }
-}
+@Beta
+public interface KpiResultRowItem {}
```

Comment:
Technically, this generated code isn't correct, but neither was the previous one, so not a breaking change.

According to the spec, this should implement `anyOf: [string, nunmber]`